include/fmt/format-inl.h

```diff
@@ -2347,7 +2356,11 @@ void fallback_format(Double d, int num_digits, bool binary32, buffer<char>& buf,
 }
 
 template <typename T>
-int format_float(T value, int precision, float_specs specs, buffer<char>& buf) {
+#ifdef FMT_HEADER_ONLY
+FMT_CONSTEXPR20
+#endif
```

Comment:
Why is it constexpr only in header-only mode?

Reply:
There is an explanation for this - https://github.com/fmtlib/fmt/issues/1895#issuecomment-863585031.

Reply:
It's actually something like this in not header-only mode: https://godbolt.org/z/a5vbjaaev

Reply:
I see. Let's at least define it in one place, i.e. something like 

```c++
#ifdef FMT_HEADER_ONLY
#define FMT_HEADER_ONLY_CONSTEXPR20 FMT_CONSTEXPR20
#endif
```

Reply:
Done